source/annotate.hs

```diff
@@ -310,18 +312,35 @@ symRepr (SymOperator val) = (text, "yellow")
           -- constants
           , (2, "t")
           , (8, "f")
+          , (14, "nil")
           -- unary operators
+          , (10, "neg")
           , (401, "dec")
           , (417, "inc")
           , (170, "mod")
           , (341, "dem")
+          , (68191693600, "pwr2")
+          , (15, "isnil")
           -- binary operators
           , (40, "div")
           , (146, "mul")
           , (365, "add")
           -- comparisons
           , (416, "lt")
           , (448, "eq")
+            -- combinators
+          -- B,C,K,W-system (https://en.wikipedia.org/wiki/B,_C,_K,_W_system)
+          , (6, "c")
+          , (5, "b")
+          , (7, "s")
+          , (1, "i")
```

Comment:
There may be a bug related to `i`. In the original dump (2c4223b) the generic decoding of the symbol was `x0`. Should it have been `:1`?

Reply:
Same with `isnil`. It's defined using `x0`, so I don't know how we'll recognize it in the wild. How did you connect it to `:15`?

Reply:
@adept, are the glyphs for `i` (m24) and `isnil` (m29) entirely new? They're not the same as each other.

`i` is a 2x2 filled-in box and `isnil` is a 3x3 filled-in box.

Based on m8, I think the smallest dimensions possible for a variable is 4x4.

Reply:
Ah, I see. This is fixed in #47.